src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpWhosInControlApplicationController.cs

```diff
@@ -74,6 +84,60 @@ public async Task<IActionResult> StartPage(Guid applicationId)
             return await AddPeopleInControl(applicationId);
         }
 
+        [HttpGet("refresh-directors-pscs")]
+        public async Task<IActionResult> RefreshDirectorsPscs(Guid applicationId)
```

Comment:
You might be able to grab UKPRN from the model in the view

Reply:
actually you'd need to grab this in the StartPage(Guid applicationId) method (we get qna application data here so it'll be in the question tags) and then through to the relevant methods & viewmodels